### PR TITLE
feat: perspective card bookmark changes

### PR DESCRIPTION
**Pull request by nitin-rachabathuni**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: https://jira.corp.adobe.com/browse/EXLM-1872

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.live/
- After: https://feat-card-bookmark--exlm--adobe-experience-league.hlx.live/

### CODE REVIEW

**Review by gauthamksn — APPROVED** [review]

**Review by ruhisingh1 — COMMENTED** [review]
Use getPathDetails(); instead of getLanguageCode()

**Review by gauthamksn — DISMISSED** [review]

File: blocks/authorable-card/authorable-card.js [hunk 1]
```diff
@@ -166,6 +55,7 @@ export default async function decorate(block) {
 
   const cardLoading$ = Promise.all(
     linksContainer.map(async (linkContainer) => {
+      // check check check
```
Comment (noormd08) [inline blocks/authorable-card/authorable-card.js:58]: Please remove this.

File: blocks/authorable-card/authorable-card.js [hunk 2]
```diff
@@ -1,120 +1,9 @@
...
-    tags: [],
-    copyLink: fullURL,
-    bookmarkLink: '',
-    viewLink: fullURL,
-    viewLinkText: placeholders[`browseCard${convertToTitleCase(type)}ViewLabel`]
-      ? placeholders[`browseCard${convertToTitleCase(type)}ViewLabel`]
-      : `View ${type}`,
-  };
-};
+import { getCardData } from '../../scripts/utils/article-utils.js';
```
Comment (noormd08) [inline blocks/authorable-card/authorable-card.js:6]: What is the change here? I dont see any need to update the authorable card changes for Perspective.

File: scripts/utils/article-utils.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
+import { getConfig } from '../scripts.js';
```
Comment (noormd08) [inline scripts/utils/article-utils.js:1]: Please rename the filename as generic.

Comment (nitin-rachabathuni) [reply scripts/utils/article-utils.js]: done Noor

File: blocks/authorable-card/authorable-card.js (hunk 1 above)
Comment (nitin-rachabathuni) [reply blocks/authorable-card/authorable-card.js]: done done

File: blocks/authorable-card/authorable-card.js (hunk 2 above)
Comment (nitin-rachabathuni) [reply blocks/authorable-card/authorable-card.js]: I lifted this code to a common place inorder to re-use Noor

**Review by noormd08 — APPROVED** [review]

**Pull request merged**